### ISSUE TITLE
Why is "Out-of-grid–total power" not updated every time?

### ISSUE THREAD

**Issue by sundsvald** [klatremis/esphome-for-deye#36]
"Out-of-grid–total power" is working as if it have a "skip_updates: 2".
Is the "skip_updates: 2" inherited from another variable?

...
- platform: modbus_controller
    modbus_controller_id: ${modbus_controller_id}
    **skip_updates: 2**
    name: "${device_type}-External CT L3 Power"
    id: ${device_type}_External_CT_L3_Power
    register_type: holding
    address: 618
    unit_of_measurement: "W"
    state_class: "measurement"
    accuracy_decimals: 0
    value_type: S_WORD     
 
  - platform: modbus_controller  
    modbus_controller_id: ${modbus_controller_id}
    name: "${device_type}-Out-of-grid–total power"
    id: ${device_type}_out_of_grid_total_power
    register_type: holding
    address: 619
    unit_of_measurement: "W"
    state_class: "measurement"
    accuracy_decimals: 0
    value_type: S_WORD
...

**Comment by fabhund** [comment]
Yes. The ESPhome modbus controller combines same beginning adresses into one call, to lower the amount of requests. The skip_updates applies to all belonging to that same start adress. It's decsribed in the ESPhome modbus controller documentation.

**Comment by sundsvald** [comment]
Thanks.
